public/v4/apps/serpbear.yml

```diff
@@ -0,0 +1,59 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        restart: always
+        image: 'towfiqi/serpbear:$$cap_serpbear_version'
+        environment:
+            NEXT_PUBLIC_BASE_URL: 'http://$$cap_appname.$$cap_root_domain'
+            USER: $$cap_serpbear_user
+            PASSWORD: $$cap_serpbear_password
+            SECRET: $$cap_serpbear_secret
+            APIKEY: $$cap_serpbear_apikey
+            SESSION_DURATION: $$cap_serpbear_session
+        caproverExtra:
+            containerHttpPort: '3000'
+        volumes:
+            - '$$cap_appname-serpbear-data:/app/data/serpbear'
+        networks:
+            - serpbear-net
```

Comment:
This is not used by CapRover, you can remove.

Reply:
ok 👌